.travis.yml

```diff
@@ -0,0 +1,82 @@
+sudo: required
+
+language: node_js
+
+node_js:
+  - '12.2'
+
+addons:
+  chrome: stable
+
+branches:
+  only:
+    - master
+    - /^greenkeeper/.*$/
+
+before_install:
+  - eval "$(ssh-agent -s)" #start the ssh agent
+  - openssl aes-256-cbc -K $encrypted_6a1e0d13907f_key -iv $encrypted_6a1e0d13907f_iv -in id_rsa.enc -out travis-deploy-key -d
+  - chmod 600 travis-deploy-key # this key should have push access
+  - ssh-add travis-deploy-key
+  - rvm install 2.5.3 && rvm use 2.5.3
+  - gem install bundler
+
+install:
+  - npm install -g gulp lerna
+  - lerna bootstrap --no-ci
+  - npm install
+  - cd packages/vapor && bundle install && cd ../../
+
+cache:
+  directories:
+    - node_modules
+    - packages/vapor/vendor/bundle
+
+script:
+ - cd packages/vapor
+ - npm run build
+ - npm run buildpages
+ - cd ../react-vapor
+ - npm run test:compile # to catch any lint or compile errors, compile tests through webpack prior to running test suites with karma (karma always exit its process with 0)
+ - npm run test
+ - npm run docs
+ - cd ../../
+
+after_script:
+  - 'if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then ./create-live-demo.sh; fi' # run this line on pull request builds only
+
+after_success:
+ - cd packages/react-vapor
+ - npm run report-coverage
+ - cd ../../
+ - mkdir -p "packages/vapor/deploy/$TRAVIS_TAG"
+ - cp -a packages/vapor/dist/. "packages/vapor/deploy/$TRAVIS_TAG"
+
+deploy:
+  - provider: surge
+    project: ./packages/react-vapor/docs/
+    domain: react-vapor.surge.sh
+    skip_cleanup: true
+
+  - provider: surge
+    project: ./packages/vapor/_gh_pages/
+    domain: vapor.surge.sh
+    skip_cleanup: true
+
+  - provider: s3
+    skip_cleanup: true
+    access_key_id: AKIAJ72WQIJQMW6TCHDA
+    secret_access_key:
+      secure: cr5PKJaLjVdIEqtwuUXgE4a0IuE9lxQRGoAkVQix9xl3hxQpXOEROUG9FUun4hERHVFGHfpwXAvjlZ2E2n4FyLk32ELwo0l27HaY0TvKd05JGx1u9JEX1YNZjyAxVLOJD7+5tlTA1I5wh5VjXIWr4STYul9sHq72gA1bIPD6xflIQDtNinod8Iic6ApZVtOG5fxOyXd7F//f1iDLTlZAfBbmeYWm0eWIywfH3XGk95i0ZuAjZheX7HM0FMgfxm0SDFCEM4IX2Inv6ML9eLLqSSSzo86QYEx0Y4aFEWBNKkiX7eMJZ30KH2S+4IvCeawJOvaN0utohaUiiPWEURo7JAfQZyoOlerIrbNv08xMveEjFg5CyRcfAtsGDsu6Z68bTQs7SRFivHxeQ1O84D0lbYQe77pK+c01h04USbfJwDz2hI0oIpqnfpOvcpcI1NH/Qvgrv6a8pe0nR4k7JH9BQ3fZtrRGeF8b4gyhVtDsb10m99VcKh7egMdoBaPFwpxY82hDso6lHfQzrec2YqcdMmtOAuEpKLqrFDiXhwm3QtFtbEPiPR8gQATv0FPWoXgDzjVTDv/ag6nxL7lQpVfHWRJURtuq9205CWkGqOHxDcUvG4zY/dGdil2FIwWpOpfp35N8bdfLaYKoyCugjgXycYF4rE5d/xXTvaiRKuBuwIQ= # https://docs.travis-ci.com/user/encryption-keys/
+    bucket: coveo-public-content
+    local-dir: packages/vapor/deploy
+    upload-dir: styleguide
+    acl: public_read
+    on:
+      tags: true
+      repo: coveo/react-vapor
+
+notifications:
```

Comment:
can we notify on slack?

Reply:
It's probably possible via a webhook but we don't have the permission to set it up